CalibTracker/SiPixelLorentzAngle/src/SiPixelLorentzAnglePCLHarvester.cc

```diff
@@ -136,6 +137,10 @@ void SiPixelLorentzAnglePCLHarvester::beginRun(const edm::Run& iRun, const edm::
   magField = &iSetup.getData(magneticFieldToken_);
```

Comment:
now there is no need to have `magField` as a data member, please clean it up.

Reply:
Done